recipes/iverilog/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "10.1" %}
+{% set git_tag = "v10_1" %}
+{% set sha256 = "f4ff7990ae9ed87fad0b486586b03f6902de0dfb9de5ecb07ef32c20487c7dc6" %}
+
+package:
+  name: iverilog
+  version: {{ version }}
+
+source:
+  fn: {{ git_tag }}.tar.gz
+  url: https://github.com/steveicarus/iverilog/archive/{{ git_tag }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    # Submitted upstream: PR: steveicarus/iverilog#116
+    # Uses /proc/self/exe at run-time to find ivl_root instead of
+    # static path at build-time.
+    - remove_static_build_path.patch
```

Comment:
👍 Thanks for doing this.

xref: https://github.com/steveicarus/iverilog/pull/116